wasm/src/lib.rs

```diff
@@ -89,24 +90,21 @@ pub fn get_library_source_content(name: &str) -> Option<String> {
 
 #[wasm_bindgen]
 pub fn get_hir(code: &str) -> String {
-    let (package, _) = compile(code);
+    let package = compile(code);
```

Comment:
I'm going to inline `compile()` and delete the function definition since it's now only called from `get_hir()` and is not as useful as the name makes it sound.